service/src/test/java/cz/fi/muni/pa165/service/BeanMapperServiceTest.java

```diff
@@ -0,0 +1,156 @@
+package cz.fi.muni.pa165.service;
+
+
+import cz.fi.muni.pa165.dto.*;
+import cz.fi.muni.pa165.entity.*;
+import cz.fi.muni.pa165.enums.Rank;
+import cz.fi.muni.pa165.enums.Role;
+import cz.fi.muni.pa165.service.config.ServiceConfig;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.BeforeClass;
+
+import javax.inject.Inject;
+
+/**
+ * Created by Erik Macej 433744 , on 21.11.16.
+ *
+ * @author Erik Macej 433744
+ */
+@ContextConfiguration(classes = ServiceConfig.class)
+public class BeanMapperServiceTest extends AbstractTestNGSpringContextTests {
+
+    @Inject
+    BeanMapperService beanMapperService;
+
+    private Visit visit1;
+    private Visit visit2;
+    private Hunter hunter1;
+    private Hunter hunter2;
+    private Mushroom mushroom;
+    private MushroomCount mushroomCount;
+    private Forest forest;
+
+    private VisitDTO visitDto1;
+    private VisitDTO visitDto2;
+    private HunterDTO hunterDto1;
+    private HunterDTO hunterDto2;
+    private MushroomDTO mushroomDto;
+    private MushroomCountDTO mushroomCountDto;
+    private ForestDTO forestDto;
+
+    @BeforeClass
+    public void prepareData(){
+        createVisits();
```

Comment:
mapper: maybe only testing of few entities should be enough
also check nulls and special cases